pkg/cassandra/util.go

```diff
@@ -76,12 +80,16 @@ func ComputeReplication(maxReplicationPerDc int, datacenters ...*cassdcapi.Cassa
 }
 
 // ComputeReplicationFromDcTemplates is similar to ComputeReplication but takes dc templates as parameters.
-func ComputeReplicationFromDcTemplates(maxReplicationPerDc int, datacenters ...api.CassandraDatacenterTemplate) map[string]int {
+func ComputeReplicationFromDcTemplates(maxReplicationPerDc int, externalDatacenters []string, datacenters ...api.CassandraDatacenterTemplate) map[string]int {
```

Comment:
Should we consider renaming the function now that it also take the `externalDatacenters` arg?

Reply:
I renamed it to `ComputeReplicationFromDatacenters`. How does that sound?

Reply:
👌